packit_service/worker/handlers/testing_farm.py

```diff
@@ -94,9 +94,21 @@ def __init__(
         self._db_trigger: Optional[AbstractTriggerDbType] = None
         self._testing_farm_job_helper: Optional[TestingFarmJobHelper] = None
 
+    def check_if_contributor_can_run_job_and_report(self, actor: str) -> bool:
+        if self.job_config.metadata.use_internal_tf and not self.project.can_merge_pr(
```

Comment:
please put the docstring here and explain a bit what is happening here and why

Reply:
It's in the parent, but I can add one here as well.